Dockerfile

```
@@ -0,0 +1,32 @@
+FROM alpine:3.12
+
+COPY ./wlc/ /wlc
+COPY ./completion/ /completion
+COPY ./setup.cfg /setup.cfg
+COPY ./setup.py /setup.py
+COPY ./wl /wl
+COPY ./LICENSE /LICENSE
+COPY ./requirements.txt /requirements.txt
+
+# This hack is widely applied to avoid python printing issues in docker containers.
+# See: https://github.com/Docker-Hub-frolvlad/docker-alpine-python3/pull/13
+ENV PYTHONUNBUFFERED=1
+
+RUN echo "**** install Python ****" && \
+    apk update && \
+    apk add --no-cache python3 && \
+    if [ ! -e /usr/bin/python ]; then ln -sf python3 /usr/bin/python ; fi && \
+    \
+    echo "**** install pip ****" && \
+    python3 -m ensurepip && \
+    rm -r /usr/lib/python*/ensurepip && \
+    pip3 install --no-cache --upgrade pip setuptools wheel && \
+    if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi && \
+    \
+    echo "**** run setup.py build script ****" && \
+    python3 setup.py build && \
+    \
+    echo "**** run setup.py install script ****" && \
+    python3 setup.py install
```

With Python image, all what is needed is to install the wlc:

```suggestion
RUN pip install -e /app/wlc
```